specification/cognitiveservices/resource-manager/Microsoft.CognitiveServices/stable/2021-04-30/cognitiveservices.json

```diff
@@ -0,0 +1,2071 @@
+      "description": "The name of SKU."
+    },
+    "Account": {
+      "description": "Cognitive Services account is an Azure resource representing the provisioned account, it's type, location and SKU.",
+      "type": "object",
+      "properties": {
+        "kind": {
+          "$ref": "#/definitions/Kind",
+          "description": "The Kind of the resource."
+        },
+        "sku": {
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/Sku"
+        },
+        "identity": {
+          "$ref": "#/definitions/Identity"
+        },
+        "systemData": {
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/systemData",
+          "readOnly": true
+        },
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "x-ms-mutability": [
+            "read",
+            "create",
+            "update"
+          ],
+          "description": "Resource tags."
+        },
+        "location": {
+          "type": "string",
+          "x-ms-mutability": [
+            "read",
+            "create"
+          ],
+          "description": "The geo-location where the resource lives"
+        },
+        "properties": {
+          "$ref": "#/definitions/AccountProperties",
+          "description": "Properties of Cognitive Services account."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/AzureEntityResource"
+        }
+      ]
+    },
+    "AccountListResult": {
+      "type": "object",
+      "properties": {
+        "nextLink": {
+          "description": "The link used to get the next page of accounts.",
+          "type": "string"
+        },
+        "value": {
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Account"
+          },
+          "description": "Gets the list of Cognitive Services accounts and their properties."
+        }
+      },
+      "description": "The list of cognitive services accounts operation response."
+    },
+    "AccountProperties": {
+      "type": "object",
+      "properties": {
+        "provisioningState": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Gets the status of the cognitive services account at the time the operation was called.",
+          "enum": [
+            "Accepted",
+            "Creating",
+            "Deleting",
+            "Moving",
+            "Failed",
+            "Succeeded",
+            "ResolvingDNS"
+          ],
+          "x-ms-enum": {
+            "name": "ProvisioningState",
+            "modelAsString": true
+          }
+        },
+        "endpoint": {
+          "type": "string",
+          "description": "Endpoint of the created account.",
+          "readOnly": true
+        },
```

Comment:
> } [](http://example.com/codeflow?start=7&length=2)

Removing internalId is a breaking change. If this is intentional, you will need a breaking change review.

Reply:
Starting breaking change request: https://msazure.visualstudio.com/One/_workitems/edit/9898180

Reply:
talked with breaking change reviewers, as suggestion by breaking change reviewers. we will bring the internalId definition back but service will not return valid value.